app/assets/javascripts/front/realization.js

```diff
@@ -0,0 +1,253 @@
+/* global zenscroll, $, jwplayer */
+/* exported Realization */
+
+class Realization {
+
+  constructor(element) {
+    this.container = element
+    this.introBtn = element.querySelector('.intro__button')
+    this.readyBtn = element.querySelector('.ready__button')
+    this.readyRemindBtn = element.querySelector('.ready__remind__link')
+    this.pausedBtn = element.querySelector('.paused__button')
+    this.pausedFinishBtn = element.querySelector('.paused__finish__link')
+    this.remindBtn = element.querySelector('.remind__button')
+    this.remindBackBtn = element.querySelector('.remind__back__link')
+    this.surveyBtns = element.querySelectorAll('.survey__button')
+    this.surveyMsg = element.querySelector('.survey__message')
+    this.surveyNextBtn = element.querySelector('.survey__next__link')
+    this.courseBtn = element.querySelector('.course__button')
+    this.thanksBtn = element.querySelector('.thanks__button')
+    this.shareBtnsContainer = element.querySelector('.share__button-wrapper')
+    this.shareBtns = element.querySelectorAll('.share__button')
```

Comment:
Could you use javascript specific selectors for these elements so that we can avoid any unintended side effects from changing the css.

So for example use `js-intro-button` for the intro button instead of `intro__button`